iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/parser/ASTVisitor.java

```diff
@@ -2005,6 +2006,28 @@ private long parseTimeValue(ConstantContext constant) {
   @Override
   public Statement visitLoadFile(IoTDBSqlParser.LoadFileContext ctx) {
     try {
+      // if sql have with, return new load sql statement
+      if (ctx.loadFileWithAttributeClauses() != null) {
+        final Map<String, String> loadTsFileAttributes = new HashMap<>();
+        if (ctx.loadFileWithAttributeClauses() != null) {
```

Comment:
I'm wondering if there's a specific reason for this nested check, or if the inner `if` statement might be unnecessary?

Reply:
It is indeed one more time, there is no need to add. 👍